pkg/reaper/deployment.go

```diff
@@ -127,6 +127,33 @@ func NewDeployment(reaper *api.Reaper, dc *cassdcapi.CassandraDatacenter, authVa
 		}
 	}
 
+	volumeMounts := []corev1.VolumeMount{}
+	volumes := []corev1.Volume{}
+	// if client encryption is turned on, we need to mount the keystore and truststore volumes
+	if reaper.Spec.ClientEncryptionStores != nil && keystorePassword != nil && truststorePassword != nil {
```

Comment:
Reminder: if we reach the conclusion that we can have a keystore without truststore, then you will need to change this block.

Reply:
After giving this a try, it turns out Cassandra won't allow providing no truststore, despite setting `require_client_auth` to `false`.
Apparently, the call to [this method](https://github.com/apache/cassandra/blob/bfab1fda882c368be0d2c9dad38cb5dbee501bfa/src/java/org/apache/cassandra/security/SSLFactory.java#L156) is made [here](https://github.com/apache/cassandra/blob/cassandra-3.11/src/java/org/apache/cassandra/security/SSLFactory.java#L61) with a hardcoded `true` for `buildTruststore`.
Cassandra then fails to start if the truststore path isn't provided.